gallery/src/components/NxCodeSnippet/NxCodeSnippet.scss

```diff
@@ -0,0 +1,5 @@
+.nx-code-snippet-custom-sizing {
+    textarea {
+        min-height: 25px;
```

Comment:
What I'm picturing is when `NxCodeSnippet` receives a defined value for `rows`, that it adds a class to itself that removes that `min-height` entirely in addition to passing the `rows` prop to the `textarea`.  So there wouldn't be a need for custom styles by downstream code such as this example.

Reply:
Thinking out loud, so I would still need an scss file and setting min-height to initial to scrap the 220px value in base styles, and add this class via classnames if rows is defined instead of having the class manually added by the user

Reply:
> Thinking out loud, so I would still need an scss file and setting min-height to initial to scrap the 220px value in base styles, and add this class via classnames if rows is defined instead of having the class manually added by the user

Well not a new scss file.  This would go in the existing `NxCodeSnippet.scss`

Reply:
sounds good, I'll start working on the changes